lesson_06/index.js

```diff
@@ -0,0 +1,171 @@
+// ДЗ 13. Реалізуйте функцію removeElement 
+
+const array = [1, 2, 3, 4, 5, 6, 7];
+
+function removeElement(array, item) {
+   array.splice((array.indexOf(item)),1)
+}
+removeElement(array, 6);
+
+console.log(array);
+
+
+
+// ДЗ 14. Реалізуйте функцію generateKey
+
+const characters = 'abcdefghijklmnopqrstuvwxyz0123456789';
+let randomKey = "";
+function generateKey(lengthKey, initialKeyData) {
+  for (i=0; i < lengthKey; i++){
+   randomKey += initialKeyData[Math.floor(Math.random() * initialKeyData.length)]
+  }
+return randomKey
+}
+
+const key = generateKey(16, characters);
+console.log(key);
+
```

Comment:
```suggestion
console.log(generateKey(16, characters));
console.log(generateKey(16, characters));
```
додайте ці два рядки сюду і Ви побачите цікавий результат)

Reply:
я зрозумів) зберігає в randomKey значення з 16 символів і потім прикожному визові аункціі додає до старого значення нове)